examples/plotting/hmi_synoptic_maps.py

```diff
@@ -47,8 +47,8 @@
 
 ###############################################################################
 # Let's also fix the plot settings
-syn_map.plot_settings['cmap'] = 'hmimag'
-syn_map.plot_settings['norm'] = plt.Normalize(-1500, 1500)
+syn_map.cmap = 'hmimag'
```

Comment:
Instead of setting `cmap` and `norm` explicitly, pass these as keyword arguments to `syn_map.plot()`